wgpu-core/src/device/resource.rs

```diff
@@ -1525,6 +1525,16 @@ impl<A: HalApi> Device<A> {
             Caps::SHADER_INT64,
             self.features.contains(wgt::Features::SHADER_INT64),
         );
+        caps.set(
+            Caps::SHADER_INT64_ATOMIC_MIN_MAX,
+            self.features
+                .contains(wgt::Features::SHADER_INT64_ATOMIC_MIN_MAX),
+        );
+        caps.set(
+            Caps::SHADER_INT64_ATOMIC_ALL_OPS,
+            self.features
+                .contains(wgt::Features::SHADER_INT64_ATOMIC_ALL_OPS),
```

Comment:
All on the name here feels weird considering you need the other feature to turn on Min/Max

Reply:
`SHADER_INT64_ATOMIC_ALL_OPS` does include min/max, though

Reply:
I made the code more explicit